common/persistence/nosql/nosqlplugin/cassandra/db.go

```diff
@@ -79,14 +77,7 @@ func (db *cdb) IsNotFoundError(err error) bool {
 }
 
 func (db *cdb) IsTimeoutError(err error) bool {
-	if err == gocql.ErrTimeoutNoResponse {
-		return true
-	}
-	if err == gocql.ErrConnectionClosed {
-		return true
-	}
-	_, ok := err.(*gocql.RequestErrWriteTimeout)
-	return ok
+	return IsTimeoutError(err)
```

Comment:
Nit: it's good to centralize the logic. Could you do the same thing for the other two?

Reply:
Sure. Forgot to do the same thing for the other two. Will update.

Reply:
Done.